source/tutorial/backend.txt

```diff
@@ -172,19 +198,15 @@ To create the webhook in your repository:
    send events for :guilabel:`Issues` and :guilabel:`Pull requests`.
 
 #. Click :guilabel:`Add webhook` to save the new webhook.
-
-.. tip::
    
-   To confirm that the webhook succesfully calls your endpoint, check
-   your :ref:`application logs <logs>` for entries with the type
-   ``Endpoint``. You can also check the `webhook's request logs
-   <https://docs.github.com/en/developers/webhooks-and-events/webhooks/testing-webhooks>`__
-   in GitHub under :guilabel:`Recent Deliveries` on the webhook's
-   settings page. Each successful request has a green checkmark next to
-   it.
+To confirm that the webhook succesfully calls your endpoint, check your :ref:`application logs <logs>` 
+for entries with the type ``Endpoint``. You can also check the `webhook's request logs
```

Comment:
you think it's worth explaining how the user can navigate to the Logs page?

Reply:
Probably doesn't hurt!